### PR TITLE
Added uptime.md

### LINKED ISSUE #461: Writing documentation for "uptime" command
Created a new command called 'uptime' and added description

### PR DESCRIPTION
**Pull request by jonniechow**
# Advanced HW 5 submission

Closes #461

Name: Jonny Chow
Uniqname: jonnny

### CODE REVIEW

File: _commands/os_x/uptime.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (AmritHariharan) [inline _commands/os_x/uptime.md]: you already have `uptime.md` in basics and shouldn't have duplicate commands.

File: _commands/basics/uptime.md [hunk 2]
```diff
@@ -3,22 +3,45 @@
...
+The system load averages for the past 1, 5, and 15 minutes
+1.91 2.04 2.13
+
+#### uptime syntax
+~~~bash
+uptime [options]
+~~~
+
+Below are some flags and their descriptions
+|     Flag     |               Description                |
```
Comment (AmritHariharan) [inline _commands/basics/uptime.md]: the table doesn't seem to be rendering correctly

File: _commands/basics/uptime.md [hunk 3]
```diff
@@ -3,22 +3,45 @@
...
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+Uptime gives a one line display for each of the following
+
+The current time:
```
Comment (AmritHariharan) [inline _commands/basics/uptime.md]: turn these into a bulleted list, also if you wanted the times shown to be on newlines that isn't showing up on the rendered file.

**Comment by cyanliu** [comment]
You've been awarded credit for Advanced homework 5 and +2 extra credit for creating your initial PR 2 days early 🎉

(You won't see this immediately reflected on Canvas, but it'll be there the next time we update advanced homework scores!)

**Pull request merged**